lib/solaar/ui/notify.py

```diff
@@ -35,7 +35,16 @@
 	from gi.repository import Notify
 
 	# assumed to be working since the import succeeded
-	available = True
+	# available = True
+
+	# This is not working on Fedora 26. If fails with:
+	# ERROR [MainThread] solaar.ui.notify: showing <Notify.Notification object at 0x7f82c2484640 (NotifyNotification at 0x556fa0fc5a40)>
+	# File "./solaar/lib/solaar/ui/notify.py", line 145, in show
+	#   n.show()
+	# Error: g-io-error-quark: Error calling StartServiceByName for org.freedesktop.Notifications: Timeout was reached (24)
+
+	available = False
+
```

Comment:
These APIs are very old, I guess that disabling them is OK especially if they cause issues like this